release/v1alpha1/types.go

```diff
@@ -88,6 +89,17 @@ type PayloadCoordinates struct {
 	ImagestreamTagName string `json:"imagestreamTagName,omitempty"`
 }
 
+// AcceptanceOverride houses the required information to force the manual Acceptance of a ReleasePayload.
+// ART, rarely, needs the ability to manually accept a Release that, for some reason or another, won't pass one or
+// more of it's blocking jobs.
+// This would be the only scenario where another party, besides the release-controller, would update a
+// ReleasePayload instance.  Upon doing so, the release-controller should see that an update occurred and make all
+// the necessary changes to formally accept the respective release.
+type AcceptanceOverride struct {
+	ForceAcceptance bool   `json:"forceAcceptance"`
+	Reason          string `json:"reason"`
```

Comment:
doc this field in particular.  Explain whether it is a human readable or not

Reply:
Added doc